src/Symfony/Component/Mime/NamedAddress.php

```diff
@@ -39,4 +41,25 @@ public function toString(): string
     {
         return $this->getEncodedNamedAddress();
     }
+
+    public static function fromString(string $string): self
+    {
+        if (false === strpos($string, '<')) {
+            return new static($string, $string);
```

Comment:
```suggestion
            return new self($string, $string);
```

The class is final anyway. 😃